metrics_reader.py

```diff
@@ -0,0 +1,70 @@
+import uuid
+from pathlib import Path
+from resim.metrics.python.metrics_writer import ResimMetricsWriter
+from resim.metrics.python.metrics import (
+    Timestamp,
+    DoubleFailureDefinition,
+    SeriesMetricsData,
+    MetricStatus,
+    MetricImportance
+)
+from resim.metrics.proto.validate_metrics_proto import validate_job_metrics
+
+def read_all_values(filename: str) -> list:
+    values = []
+    timestamps = []
+    with open(filename, 'r') as f:
+        for line in f.readlines():
+            timestamp_str, value_str = line.strip().split(',')
+            timestamps.append(Timestamp.from_string(timestamp_str))
+            values.append(float(value_str))
+    return timestamps, values
+
+def main():
+    print("Starting metrics reader...")
+    
+    try:
+        metrics_writer = ResimMetricsWriter(uuid.uuid4())
+        
+        timestamps, values = read_all_values('/tmp/resim/inputs/logs/test.log')
+        print(f"Read {len(values)} values")
+        
+        time_series = SeriesMetricsData("Random Values")
+        for t, v in zip(timestamps, values):
+            time_series.add_element(t, v)
```

Comment:
`.add_element(k, v)` isn't a method of `SeriesMetricsData`.
```suggestion
        # timestamps needs to be a list of instances of the resim Timestamp class, this is left as an exercise to the reader
        timestamps_data = SeriesMetricsData("Timestamps", series=np.array(timestamps))
        values_data = SeriesMetricsData("Random Values", series=np.array(values), index_data=timestamps_data)
```